lib/livebook_web/controllers/auth_controller.ex

```diff
@@ -24,14 +22,14 @@ defmodule LivebookWeb.AuthController do
   def index(conn, _params) do
     render(conn, "index.html",
       errors: [],
-      auth_mode: Livebook.Config.auth_mode()
+      authentication_mode: Livebook.Config.authentication().mode
     )
   end
 
   def authenticate(conn, %{"password" => password}) do
     conn = AuthPlug.store(conn, :password, password)
 
-    if AuthPlug.authenticated?(conn, :password) do
```

Comment:
We used to pass `:password` here explicitly and compare that it matches with the configured mode. I don't think it's necessary though. We have a controller plug that ensures the user is not already authenticated, so the only way they become authenticated here is if the submitted password is valid and the mode is actually password.